users/models.py

```diff
@@ -188,6 +186,19 @@ def get_project_ordered_by_last_report_creation_date(self) -> QuerySet:
             )
         ).order_by("-last_report_creation_date")
 
+    @staticmethod
+    def __parse_work_hours_and_percentage_from_queryset_to_a_dict(
```

Comment:
@maciejSamerdak please fix it in some future PR, but please use only one `_` in private methods :)

Reply:
Done